t.sentinel.import/t.sentinel.import.py

```diff
@@ -186,6 +186,13 @@
 # % answer: 1
 # %end
 
+# %option
+# % key: offset
+# % type: integer
+# % required: no
+# % description: Offset to add to the Sentinel bands to due to specific processing baseline (e.g. -1000)
```

Comment:
Should here a default be defined?

Or, better: add to the manual a note about _Sentinel-2 Processing Baseline 4.0.0 ... systematic offset..._ (from the PR description) to help the users?

Reply:
I would not add a default here, as this may change in future processing baselines, and even former baselines may still be out there. But I added a section to the user manual (which is still pretty barren, but this calls for a separate PR ;) )